test/unit/specs/components/Identity/createIdSyncs.spec.js

```diff
@@ -57,11 +46,7 @@ describe("Identity::createIdSyncs", () => {
     expect(obj[123]).toBeDefined();
     const idSyncs = createIdSyncs(config, logger, cookieJar);
     return idSyncs.process(idsToSync).then(() => {
-      const now = Math.round(new Date().getTime() / 1000 / 60 / 60); // hours
-
       obj = getControlObject();
-      expect(obj[411]).toBeDefined();
-      expect(obj[411] - now).toEqual(2);
       expect(obj[123]).toBeUndefined();
     });
   });
```

Comment:
Let's talk about how to add more tests for `createIdSyncs` sometime. You can probably add some more useful unit tests but I need to dig into the code some more in order to offer guidance. I wouldn't let it prevent this PR from being merged though.